### PR TITLE
VTCCA-3934: fixed bobby rule violation

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,9 +5,6 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 val libName = "play-conditional-form-mapping"
 
 val compileDependencies = PlayCrossCompilation.dependencies(
-  play25 = Seq(
```
Comment (PeterPerhac) [inline build.sbt:8]: I think we would still want to have this library cross-compiled for play 2.5. It's a public repo which is used by developers outside of HMRC

Comment (habeeb01) [reply build.sbt:8]: the recommended version of "sbt-play-cross-compilation" (2.0.0) no longer has support for play 2.5. Thats why I had to remove it

**Comment by darrell-plant** [comment]
LGTDz 👍

File: build.sbt (hunk 1 above)
Comment (PeterPerhac) [reply build.sbt:8]: OK, that should be fine then, people will simply have to keep downloading the older 2.5 compatible version... we stop publishing a 2.5 copy. We don't actually actively make any additions to it anyways.

**Comment by PeterPerhac** [comment]
👍  LGTPP

**Pull request merged**